### DISCUSSION TITLE
目前附录的处理方式似乎会导致目录无法正确跳转

### DISCUSSION THREAD

**Discussion by iill12** [mengchaoheng/SCUT_thesis#65] [Q&A] [answered]
首先感谢大佬的模板
我发现目前附录的处理方式似乎会导致目录无法正确跳转，
比如附录也有个2.1，那么在目录处点击附录的2.1会跳转到正文的2.1

**Comment by mengchaoheng** [comment]
ChatGPT:

在 LaTeX 中，处理附录可能会影响目录跳转的正确性。为了解决这个问题，您可以尝试以下方法：使用 hyperref 宏包： 如果您在文档中使用了 hyperref 宏包来添加超链接和书签，确保在附录部分的章节命令中正确地使用 \phantomsection。这会为目录添加一个虚拟章节，以确保目录跳转的正确性。例如，在附录章节命令之前添加 \phantomsection：\phantomsection
\chapter{附录标题}
手动添加目录项： 如果自动生成的目录项无法正确跳转到附录章节，您可以考虑手动添加目录项。您可以使用 \addcontentsline{toc}{chapter}{附录标题} 命令手动将附录标题添加到目录中，确保正确的跳转。\addcontentsline{toc}{chapter}{附录标题}
\chapter*{附录标题}
检查标签和引用： 确保在文档中正确地使用了标签和引用。如果在附录中使用了标签和引用，确保标签的定义和引用的使用正确，以便在文档中进行跳转时能够正确地定位到目标位置。通过以上方法，您应该能够解决附录导致目录无法正确跳转的问题。如果问题仍然存在，请检查文档中的其他可能影响目录跳转的因素，例如宏包冲突或错误的设置。

**Comment by iill12** [comment, marked as answer]
上面的方法尝试了，也翻来覆去问了GPT4，还是没办法完美解决，要么后续编号出问题，要么正文的节标题消失，要么书签里出现重复的节名……最后发现可以用最笨的办法，就是到PDF里面编辑跳转目标……不管怎么说好歹也是解决了

**Comment by mengchaoheng** [comment]
哦，我得好好重视这个问题，主要是附录用得比较少，同时，附录有些规定不太合逻辑。
